hack/go-version-check.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# List or check that the golang version embedded in files (specifically our Dockerfile)
+# is using a supported version of Go.
+# Run with --show to list the images the script thinks are current.
+# Run with --check (the default) to verify the Dockerfile contains a current
+# image & version tag.
+#
+
+set -e
+SCRIPT_DIR="$(cd "$(dirname "${0}")" && pwd)"
+PROJECT_DIR="$(cd "${SCRIPT_DIR}/.." && pwd)"
+WORKDIR="$(mktemp -d)"
+CONTAINER_FILES=("${PROJECT_DIR}/Dockerfile")
+GOVERSURL='https://go.dev/dl/?mode=json'
+ACTION=check
+
+
+fetch_versions() {
+    local url="$1"
```

Comment:
Shell scripts typically use one of two intent style:
- TABs (AFAIK, the most common style in the UNIX/Linux world).
- 2-spaces ([Google style](https://google.github.io/styleguide/shellguide.html#indentation))

Using 4-space (which is Python style) is fine but less common.

Reply:
4-spaces 4-eva.  ;-)
I only tolerate tabs when it's mandated by the language or there's a tool to do canonical formatting for me. :-)